### PR TITLE
some more grammer changes

### PR DESCRIPTION
**Pull request by ericfode**

### CODE REVIEW

File: src/cats/labs/writer.cljc [hunk 1]
```diff
@@ -76,16 +76,16 @@
   "The Writer type constructor.
 
   The purpose of Writer type is wrap a simple
-  function that fullfill the writer signature.
+  function that fullfills the writer signature.
```
Comment (yurrriq) [inline src/cats/labs/writer.cljc]: Writer should be capitalized here.

File: src/cats/labs/writer.cljc [hunk 2]
```diff
@@ -76,16 +76,16 @@
...
 
-  It exists just for avoid extend the clojure
-  function type because is very generic type."
+  It exists just to avoid extending the clojure
```
Comment (yurrriq) [inline src/cats/labs/writer.cljc]: Clojure should be capitalized here.

**Comment by yurrriq** [comment]
Looks good. Could you add the capitalizations I mentioned?

**Comment by ericfode** [comment]
Absolutely! Thank you for reviewing!

**Comment by ericfode** [comment]
All updated :)

**Comment by yurrriq** [comment]
Thanks. I'll review and merge later today.

**Comment by dialelo** [comment]
Thanks! :sparkles:

**Comment by yurrriq** [comment]
Thanks, @ericfode and @dialelo.

**Pull request merged**